### ISSUE TITLE
Wizard with Progressive Steps example has inaccessible radio button

### ISSUE THREAD

**Issue by jessiehuff** [patternfly/patternfly-react#4445]
**Describe the issue. What is the expected and unexpected behavior?**
When using keyboard navigation with the wizard example "Progressive Steps", only the first radio button is focusable. The second radio button is clickable but not focusable or selectable by keyboard. This is likely just an example issue.

<img width="835" alt="Screen Shot 2020-06-24 at 11 23 37 AM" src="https://user-images.githubusercontent.com/28281340/85585194-c9852e00-b60d-11ea-8492-7eb06f71230c.png">

**Is this a bug or enhancement? If this issue is a bug, is this issue blocking you or is there a work-around?**
Bug
